commands.js

```diff
@@ -820,6 +872,7 @@ var commands = exports.commands = {
 		if (targetRoom.isPrivate) {
 			return this.errorReply("Do not redirect users to private rooms. Use /invite if you must, but honestly, the room probably wants to be kept private.");
 		}
+		if (targetRoom.isPersonal) return this.sendReply("Users cannot be redirected to a personal room.");
```

Comment:
I don't think this is necessary... I wouldn't call it wrong, either, but more code complexity is bad.